h/static/scripts/util/search-text-parser.js

```diff
@@ -79,7 +79,23 @@ function shouldLozengify(phrase) {
   return true;
 }
 
-function getLozengeValuesAndIncompleteSearchTerms(queryString) {
+/**
+ * Return an array of lozenge values from the given string.
+ *
+ * @param {String} queryString A string of query terms.
+ *
+ * @returns {Object} An object with two properties: lozengeValues is an array
```

Comment:
There is a better way to document object shapes in JSDoc, search for uses of `@typedef` in the client or h codebases for examples.